src/index.ts

```diff
@@ -1 +1,19 @@
-  console.log('Happy developvding ✨')
+import express from 'express';
+const server = express();
+const port = 2000;
+server.post('/user/register', (req, res) => {
```

Comment:
```suggestion
import express from 'express';

const server = express();
const port = 2000;

server.post('/user/register', (req, res) => {
```

Давай пробелами отделять для лучшей читаемости